src/System.IO.FileSystem/tests/PortedCommon/IOInputs.cs

```diff
@@ -18,10 +18,20 @@ internal static class IOInputs
     public static bool CaseSensitive { get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) | RuntimeInformation.IsOSPlatform(OSPlatform.OSX); } }
     public static bool CaseInsensitive { get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) | RuntimeInformation.IsOSPlatform(OSPlatform.OSX); } }
 
-    // Unix values vary system to system; just using really long values here likely to be more than on the average system
-    public static readonly int MaxDirectory = 247; // Does not include trailing \0. This the maximum length that can be passed to APIs taking directory names, such as Directory.CreateDirectory, Directory.Move
-    public static readonly int MaxPath = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 259 : 10000;      // Does not include trailing \0.
-    public static readonly int MaxComponent = 255;
+    // Max path length (minus trailing \0). Unix values vary system to system; just using really long values here likely to be more than on the average system.
+    public static readonly int MaxPath = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 259 : 10000;
```

Comment:
Doesn't the current Unix implementation fail with any Unix paths longer than the Windows MAX_PATH or has that been fixed already? If it does still fail as Windows fails, then we could simplify this to 259 for both cases for a tighter test. I seem to recall you mentioning Unix long paths work normally now though.